src/coprocessor/codec/table.rs

```diff
@@ -313,29 +320,35 @@ impl RowColsDict {
         RowColsDict { value, cols }
     }
 
+    /// Returns the total count of the columns.
+    #[inline]
     pub fn len(&self) -> usize {
         self.cols.len()
     }
 
+    /// Returns whether has columns or not.
+    #[inline]
     pub fn is_empty(&self) -> bool {
         self.cols.is_empty()
     }
 
+    /// Gets a column data from its meta if key exists.
     pub fn get(&self, key: i64) -> Option<&[u8]> {
         if let Some(meta) = self.cols.get(&key) {
             return Some(&self.value[meta.offset..(meta.offset + meta.length)]);
         }
         None
     }
 
+    /// Appends a column to the row.
     pub fn append(&mut self, cid: i64, value: &mut Vec<u8>) {
         let offset = self.value.len();
         let length = value.len();
         self.value.append(value);
         self.cols.insert(cid, RowColMeta::new(offset, length));
     }
 
-    // get binary of cols, keep the origin order, return one slice and cols' end offsets.
+    // Gets binary of cols, keep the origin order, return one slice and cols' end offsets.
```

Comment:
/// Gets binary of cols, keeps the original order, and returns one slice and cols' end offsets.